<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ADSBDemodDisplayDialog</class>
 <widget class="QDialog" name="ADSBDemodDisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>289</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Display Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <layout class="QGridLayout" name="gridLayout">
      <item row="5" column="1">
       <widget class="QPushButton" name="font">
        <property name="toolTip">
         <string>Select a font for the table</string>
        </property>
        <property name="text">
         <string>Select...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="timeoutLabel">
        <property name="text">
         <string>Aircraft timeout (s)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="airportRange">
        <property name="toolTip">
         <string>Displays airports within the specified distance in kilometres from My Position</string>
        </property>
        <property name="maximum">
         <number>20000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="heliports">
        <property name="toolTip">
         <string>When checked, heliports are displayed on the map</string>
        </property>
        <property name="text">
         <string>Display heliports</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="timeout">
        <property name="toolTip">
         <string>How long in seconds after not receiving any frames will an aircraft be removed from the table and map</string>
        </property>
        <property name="maximum">
         <number>1000000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="airportSizeLabel">
        <property name="text">
         <string>Display airports with size</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="airportRangeLabel">
        <property name="text">
         <string>Airport display distance (km)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="fontLabel">
        <property name="text">
         <string>Table font</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QCheckBox" name="displayStats">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Display demodulator statistics</string>
        </property>
        <property name="text">
         <string>Display demodulator statistics</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="unitsLabel">
        <property name="text">
         <string>Units</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="airportSize">
        <property name="toolTip">
         <string>Sets the minimum airport size that will be displayed on the map</string>
        </property>
        <item>
         <property name="text">
          <string>Small</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Medium</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Large</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QCheckBox" name="autoResizeTableColumns">
        <property name="toolTip">
         <string>Resize the columns in the table after an aircraft is added to it</string>
        </property>
        <property name="text">
         <string>Resize columns after adding aircraft</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="units">
        <property name="toolTip">
         <string>The units to use for altitude, speed and climb rate</string>
        </property>
        <item>
         <property name="text">
          <string>ft, kn, ft/min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>m, kph, m/s</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ADSBDemodDisplayDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ADSBDemodDisplayDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
